### PR TITLE
When text align is default, don't add a style attribute

### PR DESCRIPTION
**Pull request by robguthrie**

### CODE REVIEW

File: packages/extension-text-align/src/text-align.ts [hunk 1]
```diff
@@ -37,9 +37,13 @@ export const TextAlign = Extension.create<TextAlignOptions>({
...
-            renderHTML: attributes => ({
-              style: `text-align: ${attributes.textAlign}`,
-            }),
+            renderHTML: attributes => {
+              if (attributes.textAlign == this.options.defaultAlignment){
+                return {}
+              } else {
+                return { style: `text-align: ${attributes.textAlign}` }
+              }
+            },
```
Comment (zcuric) [inline packages/extension-text-align/src/text-align.ts:46]: Use `===` for comparison. You don't need `else`. 

```suggestion
            renderHTML: attributes => {
              if (attributes.textAlign === this.options.defaultAlignment) {
                return {};
              }
               
               return { style: `text-align: ${attributes.textAlign}` }
            },
```

**Review by zcuric — APPROVED** [review]

**Review by zcuric — CHANGES_REQUESTED** [review]
Good idea, but please address these suggestions.

**Comment by robguthrie** [comment]
thanks for the tips

**Comment by zcuric** [comment]
@philippkuehn LGTM :)

**Comment by klaasgeldof** [comment]
Was running into issues with TextAlign today. Even if you don't touch the align buttons, it always sets a style attribute of `text-align: left`. Then when you use the value in your HTML, for example like this:

```html
<div class="text-center">
    {{ $value }}
</div>
```

...the text is always left aligned.

This PR should fix this.

**Pull request merged**